spec/v1/envelope.md

```diff
@@ -1,18 +1,41 @@
 # Envelope layer specification
 
-Version: [DSSE v1.0]
-
-The Envelope is the outermost layer of the attestation, handling
-authentication and serialization.
+The Envelope is the outermost layer of the attestation, handling  serialization
+and authentication (via digital signatures).
 
 ## Schema
 
-The format and protocol are defined per [DSSE v1.0].
+The RECOMMENDED format and protocol for Envelopes are defined per [DSSE v1.0].
+Producers MAY use other signature methods and formats that meet the [ITE-5]
+specification:
+
+-   MUST support the inclusion of multiple signatures in a single envelope
+-   SHOULD include an authenticated payload type
+-   SHOULD avoid depending on canonicalization for security
+-   SHOULD support a hint indicating what signing key was used, i.e., a KEYID
+-   SHOULD NOT require the verifier to parse the payload before verifying
+-   SHOULD NOT require the inclusion of signing key algorithms in the signature
+
+### Alternative Envelope schemas
+
+-   The [Sigstore Bundle], while [supporting DSSE], is not currently [ITE-5]
+    compliant because it requires a _single signature_ in the envelope.[^1]
+-   The [COSE_Sign] structure is [ITE-5] compliant, whereas the `COSE_Sign1`
+    format that supports only one signer is NOT compliant.
 
 ## Fields
 
-The in-toto Attestation Framework has the following requirements for the
-standard DSSE fields.
+The in-toto Attestation Framework has the following general field requirements
+for an Envelope:
+
+-   `signature` (or equivalent) is REQUIRED and MUST be defined as an array.
+-   A `keyid` (or equivalent) SHOULD be included for each signing key used.
+-   `payload` (or equivalent) SHOULD be included and contain the attestation
+    data that was signed.
+-   `payloadType` (or equivalent) SHOULD be signed along with the `payload`.
+
+In addition, the Envelope spec has the following specific requirements for the
```

Comment:
should this say more about how payloadType is structured for other envelopes? i.e., that it communicates this is an in-toto payload and the serialized format?

Reply:
Good point, we probably should. 

And maybe this is stricter than ITE-5, but I've also been wondering if that SHOULD should be a MUST? As in: "`payloadType` (or equivalent) MUST be signed along with the `payload`"

Reply:
Are we blocking only on this?

Reply:
Yes, sorry for the delay on this.

Reply:
No worries! I asked only because I wanted to know whether either or both of you needed help with this :)

Reply:
alright, PTAL https://github.com/in-toto/attestation/pull/431/commits/4c0ae5eedd40a413ed886ecc5189174b57365cc2 . 
@adityasaky @trishankatdatadog

Reply:
lgtm